### PR TITLE
Fix naming convension and Dart Analysics warnings

### PR DESCRIPTION
**Pull request by matthiasbruns**
https://dart.dev/guides/language/effective-dart/style

and removing warnings from dart analysis

### CODE REVIEW

File: lib/network/repository_notifications.dart [hunk 1]
```diff
@@ -50,10 +50,10 @@ class ResponseNotificationItem {
   String get address => _address;
 
   ResponseNotificationItem(
-      this._Id, this._userId, this._timestamp, this._address);
+      this._id, this._userId, this._timestamp, this._address);
 
   ResponseNotificationItem.map(dynamic obj) {
-    _Id = obj["id"];
+    _id = obj["id"];
```
Comment (matthiasbruns) [inline lib/network/repository_notifications.dart:56]: the string "id" is different then "Id" from toMap - is this a bug?

File: test/widget_test.dart [hunk 2]
```diff
@@ -5,26 +5,23 @@
...
-    expect(find.text('1'), findsNothing);
-
-    // Tap the '+' icon and trigger a frame.
-    await tester.tap(find.byIcon(Icons.add));
-    await tester.pump();
-
-    // Verify that our counter has incremented.
-    expect(find.text('0'), findsNothing);
-    expect(find.text('1'), findsOneWidget);
+//    // Build our app and trigger a frame.
```
Comment (tsjamm) [inline test/widget_test.dart]: Curious about these being commented out?

Comment (matthiasbruns) [reply test/widget_test.dart]: the test does not work anyway and it marks the files in my IDE as red

Comment (tsjamm) [reply test/widget_test.dart]: Okay. Can we add a TODO or a FIXME so that someone can pick it up later as required.

Comment (matthiasbruns) [reply test/widget_test.dart]: Sure why not. @SAGARSURI and I would have added tests after a few more housekeeping PRs anyway for the whole app

**Comment by matthiasbruns** [comment]
@tsjamm review pls - merging is painful :D

File: lib/network/repository_notifications.dart (hunk 1 above)
Comment (oianmol) [reply lib/network/repository_notifications.dart:56]: I think it's autogenerated from json. It's good that you renamed it.

**Pull request merged**